src/v2/view-builder/views/google-authenticator/EnrollGoogleAuthenticatorController.js

```diff
@@ -0,0 +1,52 @@
+import { loc, createButton } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+import EnrollGoogleAuthenticatorBarcodeView from './EnrollGoogleAuthenticatorBarcodeView';
+import EnrollAuthenticatorMannuallySetupView from './EnrollAuthenticatorMannuallySetupView';
+import FormType from '../../../../util/FormType';
+
+
+export default BaseForm.extend({
+  hasSavingState: false,
+  title () {
+    return null;
+  },
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    this.add(EnrollGoogleAuthenticatorBarcodeView, {options: {model: this.model}});
+    this.add(EnrollAuthenticatorMannuallySetupView);
+    this.addInput({
+      label: false,
+      className: 'shared-secret',
+      type: 'text',
+      placeholder: this.options.appState.get('currentAuthenticator').contextualData.sharedSecret,
+      disabled: true,
+    });
+  },
+  className: 'oie-enroll-google-authenticator',
+
+  modelEvents: {
+    'triggerBarcode': '_triggerBarcode'
+  },
+
+  render () {
+    BaseForm.prototype.render.apply(this, arguments);
+    this.$('.oie-enroll-google-authenticator-manual-setup').hide();
```

Comment:
want to understand why we need to do this and find a better way to show/hide them.. Is it a toggle ?